cvhtml.html

```diff
@@ -0,0 +1,52 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>    
+        <main>        
+            <h1>Yukhimiuk Ilya</h1>
+            <hr>
+            <h2>Summary:</h2>
+            <hr>
+            <p>Hi, my name is Ilya and i’m gonna be developer. I have a lot of time and potential due to my age (only 20 years). I spend a lot of time with computer and already have good knowledge. I have a lot of friends that already are developers and them give me a lot of motivation and tips how to be better, so i will end this school. </p>
+            <h2>Contacts:</h2>
+            <hr>
+            <ul> 
+                <li>Phone: +375(33) 638-18-80</li>
+                <li>Skype: +375(33) 638-18-80</li>   
+                <li>GitHub: https://github.com/pbl6asoad</li>
+                <li>E-mail: pbl6a.soad@gmail.com</li>
+            </ul>
+            <hr>
+            <h2>Skills:</h2>
+            <hr><img src="hands.png" alt="">
+            <p>That’s skills that i know or studying:</p>
+            <ul> 
+                <li><input type="checkbox" checked>HTML</li>
+                <li><input type="checkbox" checked>CSS</li>   
+                <li><input type="checkbox">GitHub</li>
+                <li><input type="checkbox">JavaScript</li>
+            </ul>
+            <h2>Education:</h2>
+            <hr>
+            <table> 
+                <tr>    
+                    <th>Timeline</th>
+                    <th>Type of education</th>
+                </tr>
+                <tr>    
+                    <td>2015 - present</td>
+                    <td>Four-year student, Faculty of Engineering and Economics, Belarusian State University of Informatics and Radioelectronics, Minsk, Belarus</td>
+                </tr>
+            </table>
+            <h2>English-level:</h2>
+            <hr>
+            <p>My English-level is about B1. I haven't passed any courses. All my practice is speaking with foreigners sometimes, but they don't know English very well aswell. I've got 9 for the English exam and received 60/100 for entrance exams.</p>   
+        </main>      
+    
+    </body>
+</html>

```

Comment:
I know, that this task was done without any  specific requirements, but further you should use css classes for styling. Also, for naming use some methodologies, for example BEM:  https://ru.bem.info/methodology/quick-start/